students/laud/session02/lightning.py

```diff
@@ -0,0 +1,35 @@
+import random
+import datetime
+
+names = '''
+Arasim  David
+Chen    Qi
+Cheung  Jackie
+Degraw  Shawn D
+Guo Ying
+Jakubiak    Jason
+Meese   Tim
+Minavi  Alexander
+Morrison    Isaac
+Nalla   Arun K
+Nguyen  Brandon Q
+Schirra Rachel Elizabeth
+Song    Yushu
+Tetteh  Laud N
```

Comment:
I should have mentioned specifically not to copy/create this file in your directory... It's not good to show all these names in public, so please remove this file from your directory. You can get the redacted source file from `examples` directory already.

Reply:
@hosungs Thanks for pointing that out. I remove it.